app/assets/stylesheets/marking_schemes.scss

```diff
@@ -0,0 +1,16 @@
+@import 'common/core';
+
+table {
+  td {
+    padding-bottom: 1em;
+    padding-right: 1em;
+  }
+}
+
+h2 {
+  padding-bottom: 1em;
+}
+
+.new_ms, .edit_ms {
```

Comment:
Selector `new_ms` should be written in lowercase with hyphens<br>Selector `edit_ms` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line